packages/amplify_core/lib/src/types/models/auth_rule.dart

```diff
@@ -90,7 +91,7 @@ class AuthRule {
         groupsField: map['groupsField'],
         provider: map['provider'],
         operations: List<ModelOperation>.from(
-            map['operations']?.map((x) => ModelOperation.values[x])));
+            map['operations']?.map((dynamic x) => ModelOperation.values[x])));
```

Comment:
nit: `dynamic` probably is unnecessary here as the analyzer infers `x` as `dynamic` already?

Reply:
discussed offline, but these changes were needed to satisfy the analysis linter as the files are now subject to analysis in core (but not datastore, yet).